meetup/api.py

```diff
@@ -24,9 +24,13 @@ class MeetupClient(object):
     rate_limit_reset = 1
     last_response_time = None
 
-    def __init__(self, api_key):
+    request_headers = {}
```

Comment:
I think it would be better to remove this default value and just set the value in the constructor.